content/post/commonifying-llvm-in-nixpkgs/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Commonifying LLVM in Nixpkgs
+description: How I was able to reduce complexity and technical debt in Nixpkgs for LLVM.
+date: 2024-06-08 18:16:00 +0800
+---
+
+In programming, technical debt is a real thing. It often occurs with legacy code or
```

Comment:
"technical debt is a real thing" reads somewhat bizarre, maybe rephrasing this to like "Technical debt is a common hurdle in large projects, if unmanaged it can significantly impede improvements to a project"

Reply:
Alright, is the way it is now better?